docs/src/emulation.md

```diff
@@ -425,7 +425,7 @@ Allow notifications for a specific domain.
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 const config = {
   use: {
-    permissions: ['notifications'], {origin: 'https://skype.com'},
+    permissions: ['notifications'],
```

Comment:
This is now not an example for "Allow notifications for a specific domain." :)

Reply:
maybe we should add a beforeAll with context.grandPermissions instead?

Reply:
Sure!